### DISCUSSION TITLE
Problems with the https://photon.komoot.io/ API

### DISCUSSION THREAD

**Discussion by Intre2023** [komoot/photon#730] [General]
Greetings,
I have a problem with the https://photon.komoot.io/ site API. I integrated the above APIs into some web services and everything worked fine from our server address (ip 94.177.185.130), but suddenly I stopped getting a response. So I've done some tests both locally and on another server and everything works fine, what could it be due to? In the hope that you can show me some solutions to my problem, 

Best regards

**Comment by mtmail** [comment]
From the bottom of the website (https://photon.komoot.io/) "You can use the API for your project, but please be fair - extensive usage will be throttled. We do not guarantee for the availability and usage might be subject of change in the future."
